packages/paste-website/src/components/Roadmap/Roadmap.tsx

```diff
@@ -40,13 +40,14 @@ const Roadmap: React.FC<RoadmapProps> = ({data}) => {
     <Box width="100%">
       <Stack orientation="vertical" spacing="space190">
         {data.map((release) => {
-          const releaseSlug = `#${slugify(release.release)}`;
```

Comment:
CAUGHT! 🐛 

And I think it was a good idea to do the slug as not having the `#` and add that into where it was relevant 😎